simulation/vis_simulate.pro

```diff
@@ -19,15 +20,16 @@ FUNCTION vis_simulate,obs,status_str,psf,params,jones,skymodel,file_path_fhd=fil
   
   IF N_Elements(status_str) GT 0 THEN IF Min(status_str.vis_ptr[0:n_pol-1]) EQ 0 THEN recalculate_all=1
   IF N_Elements(recalculate_all) EQ 0 THEN recalculate_all=1
-  
+ 
+ 
   ;Construct model visibilities. Start by building a model u-v-f cube
   if keyword_set(include_catalog_sources) then begin
     catalog_source_array=generate_source_cal_list(obs,psf,catalog_path=catalog_file_path,_Extra=extra)
     if n_elements(source_array) gt 0 then source_array = [source_array, catalog_source_array] else source_array = catalog_source_array
   endif    
   n_sources=N_Elements(source_array)
   print, 'n_sources: '+string(n_sources)
-  skymodel=fhd_struct_init_skymodel(obs,source_list=source_array,catalog_path=catalog_file_path,diffuse_model=diffuse_model, return_cal=0,_Extra=extra)
+  skymodel=fhd_struct_init_skymodel(obs,source_list=source_array,catalog_path=catalog_file_path,diffuse_model=diffuse_model, beam_threshold=beam_threshold, return_cal=0,_Extra=extra)
```

Comment:
I don't think `beam_threshold` is defined anywhere else, so it shouldn't be passed in.